addons/stock_account/models/stock_quant.py

```diff
@@ -73,8 +73,8 @@ def _apply_inventory(self):
             else:
                 super(StockQuant, inventories)._apply_inventory()
 
-    def _get_inventory_move_values(self, qty, location_id, location_dest_id, package_id=False, package_dest_id=False):
-        res_move = super()._get_inventory_move_values(qty, location_id, location_dest_id, package_id, package_dest_id)
+    def _get_inventory_move_values(self, qty, location_id, location_dest_id, package_id=False, package_dest_id=False, product_id=False, lot_id=False, owner_id=False):
+        res_move = super()._get_inventory_move_values(qty, location_id, location_dest_id, package_id, package_dest_id, product_id, lot_id, owner_id)
```

Comment:
Write a test to check the valuation is correctly updated in case `owner_id` is updated